ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -277,3 +277,47 @@ def test_hamming_window(
         beta=beta,
         dtype=dtype,
     )
+
+
+@handle_test(
+    fn_tree="functional.experimental.tril_indices",
+    n_rows=helpers.ints(min_value=0, max_value=10),
+    n_cols=st.none() | helpers.ints(min_value=0, max_value=10),
+    k=helpers.ints(min_value=-11, max_value=11),
+    dtypes_x=helpers.dtype_and_values(available_dtypes=("int32", "int64")),
+)
+def test_tril_indices(
+    *,
+    n_rows,
+    n_cols,
+    k,
+    num_positional_args,
+    as_variable,
+    # with_out,
+    native_array,
+    container_flags,
+    instance_method,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+    dtypes_x,
+):
+    dtypes, _ = dtypes_x
+    helpers.test_function(
+        input_dtypes=dtypes,
+        ground_truth_backend=ground_truth_backend,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        container_flags=container_flags,
+        instance_method=instance_method,
```

Comment:
The test are failing due to generated as `instance_method=True` which means we should test the function as an instance method, which is something we shouldn't do here, as there's no instance method of `tril_indicies`, we should change this to `instance_method=False`.